app/src/main/java/com/example/areumdap/Data/repository/ChatReportRepositoryImpl.kt

```
@@ -0,0 +1,27 @@
+                size = size
+            )
+
+//            HTTP 레벨 성공 체크
+            if (!res.isSuccessful) {
+                error("HTTP ${res.code()} ${res.message()}")
+            }
+
+            val body = res.body() ?: error("응답 body가 비어있음")
+
+            if (!body.isSuccess || body.data == null) {
+            }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyIfBlock)